doc/default.nix

```diff
@@ -6,7 +6,8 @@ in
 pkgs.stdenv.mkDerivation {
   name = "nixpkgs-manual";
 
-  buildInputs = with pkgs; [ pandoc libxml2 libxslt zip jing  xmlformat ];
+  buildInputs = with pkgs; [ pandoc libxml2 libxslt zip jing xmlformat
+    python36 nix ];
```

Comment:
python3 is sufficient. Note that soon I intend to make python37 the default, and this code will still work fine then.